packages/node/src/types.ts

```diff
@@ -11,6 +11,9 @@ export interface BaseNodeOptions {
   /** Sets an optional server name (device name) */
   serverName?: string;
 
+  /**  */
+  includeStackLocals?: boolean;
```

Comment:
Do we really want to include an option here? Can users not enable this feature by adding the integration?

Reply:
Maybe for now we put it under the experiments flag?